Release System Charts for 2.5.10

thedadams opened the pull request:
https://github.com/rancher/rancher/issues/33528

cmurphy committed:
Add version 1.0.200 of gke-operator

cmurphy committed:
Add version 1.0.200 of gke-operator

cmurphy committed:
Update gke-operator to 1.0.2-rc1

cmurphy committed:
Update gke-operator to 1.0.2-rc1

cmurphy committed:
Merge pull request #463 from cmurphy/gke-operator-1.0.2-rc1

Update gke-operator to v1.0.2-rc1

Commit:
Add additional trusted CAs to deployment

When Rancher is installed on a cluster behind a proxy, gke-operator
needs to trust the certificate from the proxy. Therefore, it is passed
to the deployment as a secret that is setup by the user as per the
documentation.

Commit:
Add additional trusted CAs to deployment

When Rancher is installed on a cluster behind a proxy, gke-operator
needs to trust the certificate from the proxy. Therefore, it is passed
to the deployment as a secret that is setup by the user as per the
documentation.

jiaqiluo committed:
fix the min_rancher_version on monitoring and gke-operator
The reason why this is required is due to the custom ordering of semver used by the Rancher dependency in https://github.com/mcuadros/go-version/blob/master/compare.go#L13

jiaqiluo committed:
fix the min_rancher_version on monitoring and gke-operator
The reason why this is required is due to the custom ordering of semver used by the Rancher dependency in https://github.com/mcuadros/go-version/blob/master/compare.go#L13

Commit:
Merge pull request #472 from thedadams/gke-additional-ca

dramich committed:
Merge pull request #476 from jiaqiluo/fix-min-rancher-version

fix the min_rancher_version on monitoring and gke-operator

Commit:
Add eks-operator 1.0.801

Commit:
Add eks-operator 1.0.801

Commit:
Add additional CA to eks-operator

Commit:
Add additional CA to eks-operator

Commit:
Merge pull request #438 from rancher/bhealey-eks-operator-additional-ca

cmurphy committed:
Bump gke-operator image to 1.1.1-rc3

cmurphy committed:
Bump gke-operator image to 1.1.1-rc3

cmurphy committed:
Merge pull request #481 from cmurphy/gke-operator-1.1.1-rc3-dev-v2.5

Bump gke-operator image to 1.1.1-rc3

cmurphy committed:
Update gke-operator

cmurphy committed:
Update gke-operator

cmurphy committed:
Merge pull request #488 from cmurphy/gke-private-cluster-fix-v2.5

[2.5] Update gke-operator

Commit:
Add eks-operator 1.0.1000 dir

Commit:
Add eks-operator 1.0.1000 dir

Commit:
Bump eks-operator to 1.0.10-rc1

Commit:
Bump eks-operator to 1.0.10-rc1

cmurphy committed:
Sync gke-operator 1.0.200 chart with 2.5.9

Forward-port 2.5.9 changes to 1.0.200, move old 1.0.200 to 1.0.300 for
2.5.x.

cmurphy committed:
Sync gke-operator 1.0.200 chart with 2.5.9

Forward-port 2.5.9 changes to 1.0.200, move old 1.0.200 to 1.0.300 for
2.5.x.

cmurphy committed:
Bump gke-operator to 1.1.1-rc7

cmurphy committed:
Bump gke-operator to 1.1.1-rc7

Commit:
Merge pull request #493 from thedadams/bump-eks-v1.0.10-rc1

cmurphy committed:
Merge pull request #500 from cmurphy/fix-gke-operator-charts-2.5

[2.5] Fix gke-operator charts

cmurphy committed:
Sync gke-operator 1.0.300 from 2.5

1.0.200 is removed as it was never released.

cmurphy committed:
Sync gke-operator 1.0.300 from 2.5

1.0.200 is removed as it was never released.

cmurphy committed:
Copy gke-operator package to 1.0.400

cmurphy committed:
Copy gke-operator package to 1.0.400

cmurphy committed:
Merge pull request #508 from cmurphy/gke-operator-1.0.3-2.5

Sync gke-operator 1.0.200 package from 2.5.9

cmurphy committed:
Update gke-operator to 1.1.1-rc10

cmurphy committed:
Update gke-operator to 1.1.1-rc10

cmurphy committed:
Merge pull request #510 from cmurphy/gke-operator-1.1.1-rc10

Update gke-operator to 1.1.1-rc10

Commit:
added rancher-logging 0.2.5

Commit:
added rancher-logging 0.2.5

Commit:
bumped loggging images

Commit:
bumped loggging images

paynejacob committed:
Merge pull request #514 from paynejacob/25logging-bump

25logging bump

Commit:
Bump eks-operator to 1.0.10-rc2

In addition to bumping the image version, the securityContext is added to
support adding additional CAs when necessary.

Commit:
Bump eks-operator to 1.0.10-rc2

In addition to bumping the image version, the securityContext is added to
support adding additional CAs when necessary.

Commit:
Add securityContext to gke-operator

The securityContext is necessary when adding additional CAs.

Commit:
Add securityContext to gke-operator

The securityContext is necessary when adding additional CAs.

Commit:
Merge pull request #531 from thedadams/bump-eks-1.0.10-rc2

Commit:
Merge pull request #530 from thedadams/gke-security-policy

Commit:
Put additional cert in /etc/rancher/ssl

In order for `c_rehash` to find the additional certificate, the cert
file needs to be placed in /etc/rancher/ssl.

Commit:
Put additional cert in /etc/rancher/ssl

In order for `c_rehash` to find the additional certificate, the cert
file needs to be placed in /etc/rancher/ssl.

Commit:
Merge pull request #532 from thedadams/fix-kev2-cert

paynejacob committed:
udpated fluentd images

paynejacob committed:
udpated fluentd images

paynejacob committed:
Merge pull request #535 from paynejacob/issue-34770

fluentd image bump

Commit:
Rename 1.1.200 to 1.1.100 for gke-operator

Commit:
Rename 1.1.200 to 1.1.100 for gke-operator

Commit:
Downgrade gke-operator to 1.1.1

Commit:
Downgrade gke-operator to 1.1.1

Commit:
Merge pull request #542 from thedadams/down-gke-1.1.1

Commit:
Bump eks-operator to 1.0.10

Commit:
Bump eks-operator to 1.0.10

Commit:
Merge pull request #543 from thedadams/bump-eks-v1.0.10

cbron commented on charts/rancher-gke-operator/1.0.300/Chart.yaml:
```diff
@@ -0,0 +1,8 @@
+home: https://github.com/rancher/gke-operator
+sources:
+  - "https://github.com/rancher/gke-operator"
+version: 1.0.300
```
This is already live ? And not supposed to be here according to: https://github.com/rancher/rancher/issues/33528
https://github.com/rancher/system-charts/tree/release-v2.5/charts/rancher-gke-operator/1.0.300

thedadams replied:
I messaged you about this. This is live. The reason it is showing up is a little weird, but I am sure it has something to do with the fact that this PR for 2.5.9 came from `dev-v2.5.9` branch instead of `dev-v2.5` branch.

cbron replied:
Can you try pulling in/rebasing release to this branch to see if it goes away.

thedadams replied:
I created a PR with this rebase: https://github.com/rancher/system-charts/pull/544

cbron commented on charts/rancher-logging/0.2.5/Chart.yaml:
```diff
@@ -0,0 +1,12 @@
+description: Rancher logging helm chart to support logging function in rancher
+icon: https://raw.githubusercontent.com/fluent/fluentd-docs/master/public/logo/Fluentd_square.png
+name: rancher-logging
+version: 0.2.5
```
This is also not in "new charts" issue: https://github.com/rancher/rancher/issues/33528
I believe we talked about this, but can someone track down.

thedadams replied:
Yes, @paynejacob reviewed and said these changes are good to go in here. I will update new charts issue.

cbron commented on charts/rancher-monitoring/v0.2.2/questions.yml:
```diff
@@ -1 +1 @@
-rancher_min_version: 2.5.8-rc1
+rancher_min_version: 2.5.8-patch1
```
curious that we are moving to patch for everything

Commit:
Merge pull request #544 from thedadams/rebase-on-release

cbron reviewed (dismissed):
lgtm. only problem is we literally have 3 different min versions: rc1, patch1, alpha1. Technically this should cause any problems, we didn't create any special 2.5 releases that I'm aware of, but we should standardize eventually. 
Will wait to merge until final sign off.

jiaqiluo commented on charts/rancher-eks-operator/1.0.800/questions.yaml:
```diff
@@ -1 +1,2 @@
 rancher_min_version: 2.5.6-alpha1
+rancher_max_version: 2.5.9
```
This is the only change outside those 3 new charts.

jiaqiluo reviewed (dismissed):
LGTM

Files match the new chart in the issue. 
``` 
rancher-eks-operator | 1.0.1000  
rancher-gke-operator | 1.1.100  
rancher-logging | 0.2.5
```

aiyengar2 reviewed (commented):
- [x] All charts in this PR are within https://github.com/rancher/rancher/issues/33528 and no additional charts are in this PR?

- rancher-eks-operator
  - [x] Latest version is `1.0.1000`?
  - [x] rancher_max_version of previous chart is `2.5.9`?
  - [x] rancher_min_version of latest chart is `2.5.10-alpha1`?
- rancher-gke-operator
  - [x] Latest version is `1.1.100`?
  - [ ] rancher_max_version of previous chart is `2.5.9`?
    - **Is this required? Currently, there is no rancher_max_version for the previous chart.**
  - [x] rancher_min_version of latest chart is `2.5.10-alpha1`?
    - **Inconsistent: currently is `2.5.10-patch1`**
- rancher-logging
  - [x] Latest version is `0.2.5`? 
  - [ ] rancher_max_version of previous chart is `2.5.9`?
    - **Is this required? Currently, there is no rancher_max_version for the previous chart.**
  - [x] rancher_min_version of latest chart is `2.5.10-alpha1`?
    - **Inconsistent: current version is `2.5.10-rc1` and the `questions.yaml` is named `questions.yml` instead.**

`rancher_min_version` and older `rancher_max_version` are inconsistent. IMO, we should standardize on having the latest chart have `rancher_min_version: 2.5.10-alpha1` and having the previous latest chart have `rancher_max_version: 2.5.9`, similar to what was established in https://github.com/rancher/system-charts/pull/527.

@cbron should we do a PR to fix this or leave it as is? Functionally, all charts are in a state where they will still work, but we should clean up / standardize IMO.

If it is decided that no changes are required, this PR LGTM.

Commit:
Standardize min rancher versions for new charts

In 2.5.10, there are three charts that are introduced. The min
versions for these charts are inconsistent. This change fixes
these inconsistencies.

Commit:
Merge pull request #545 from thedadams/min-max-version-fixes

cbron commented on charts/rancher-eks-operator/1.0.1000/questions.yaml:
```diff
@@ -0,0 +1 @@
+rancher_min_version: 2.5.10-alpha1
```
I'm not sure questions.yaml actually works, as opposed to questions.yml. But as these are both kev2 operators it doesn't actually matter.

cbron reviewed (approved):
lgtm

aiyengar2 reviewed (approved):
LGTM
